gateway/api/session/session.go

```diff
@@ -783,3 +784,278 @@ func Kill(c *gin.Context) {
 
 	c.JSON(http.StatusNoContent, nil)
 }
+
+func createApprovedReview(ctx *storagev2.Context, session *models.Session, conn *models.Connection, user *models.User, sessionInfo *openapi.SessionApproved) error {
+	var accessDuration time.Duration
+	if sessionInfo.Duration != nil {
+		accessDuration = time.Duration(*sessionInfo.Duration) * time.Second
+
+		if accessDuration.Hours() > 48 {
+			return fmt.Errorf("jit access input must not be greater than 48 hours")
+		}
```

Comment:
do we have this constraint generally or is it something added here for some reason?